packages/component-library/src/LandingPage/DataList.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router';
+
+import { css } from 'emotion';
+
+import localSVG from '../../assets/local/local.svg';
+import usaSVG from '../../assets/country/usa.svg';
+
+const cardsWrapper = css`
+  display: flex;
+  justify-content: space-around;
+  padding: 0;
+  margin: 0;
+  flex-wrap: wrap;
+`;
+const card = css`
+  font-family: 'Rubik';
+  position: relative;
+  background-color: transparent;
+  font-size: 2vw;
+  flex: 0 0 auto;
+  width: 100%;
+  padding: 12px 0;
+  box-sizing: border-box;
+  text-align: left;
+  margin: 6px 0px;
+  transition: all .3s ease-in-out;
+
+  :not(:last-child):before {
+    content: '';
+    height: 40px;
+    border-right: 1px solid #ec485b;
+    top: 82%;
+    left: 50px;
+    position: absolute;
+  }
+`;
+const cardTextWrapper = css`
+  display: inline-block;
+  position: relative;
+  vertical-align: top;
+  width: 50%;
+`;
+const eyebrowStyle = css`
+  display: block;
+  font-family: 'Rubik';
+  font-size: 14px;
+  color: #EE495C;
+`;
+const locationTitle = css`
+  display: block;
+  font-family: 'Rubik';
+  font-size: 18px;
+  color: white;
+`;
+
+const iconWrapper = css`
+  display: inline-block;
+  width: 100px;
+  margin: 0;
+  margin-right: 24px;
+  text-align: center;
+`;
+const locationResult = css`
+  color: white;
+  font-family: 'Rubik',sans-serif;
+  font-size: 14px;
+  width: 180px;
+`;
+const whyStyle = css`
+  color: #ef495c;
+
+  a {
+    border: none;
+    text-decoration: none;
+
+    :hover {
+      text-decoration: underline;
+    }
+  }
+`;
+
+const DataList = ({ city, state }) => {
+  const missingStates = ['AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DE'];
```

Comment:
How did we end up with missing states?

Reply:
I think some of the states were missing from the icon package that we're using 🤷‍♂️ i'm hoping to find some replacements soon.

Reply:
Yeah, there were totally missing shapes... there were also shapes that were mislabeled. It's a good thing that we knew what Maryland looked like.